### PR TITLE
add basic rendering of filters from swagger spec

### PR DESCRIPTION
**Pull request by jamesgorrie**
A start on how we might expose unfiltered results to internal people.
Need to create a toggle for this too.

### CODE REVIEW

File: common/views/components/Filters/Filters.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+const Filters = ({
+  schema,
+  values
+}: Props) => {
+  const checkboxGroupParams = schema.parameters.filter(
+    param => param.in === 'query' && 'enum' in param.schema
+  );
+  return (
+    <div>
```
Comment (davidpmccormick) [inline common/views/components/Filters/Filters.js]: Could be Fragment? Not that it matters.

File: common/views/components/Filters/Filters.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+              <span>{param.schema.enumTitles[i]}</span>
+              <input
+                type='checkbox'
+                value={value}
+                name={param.name}
+                checked={
+                  values[param.name] &&
+                  values[param.name].indexOf(value) !== -1
+                }
+                onChange={(e) => console.info(e)} />
```
Comment (davidpmccormick) [inline common/views/components/Filters/Filters.js]: Is this to stay?

Comment (jamesgorrie) [reply common/views/components/Filters/Filters.js]: It was for debugging, the filters don't actually do anything, they just sit there.

**Comment by jamesgorrie** [comment]
Closing as it's a nice idea, but feel it wouldn't really work.

**Pull request closed without merging**